templates/security.html.twig

```diff
@@ -52,6 +52,33 @@
                             </label>
                             <div class="input-group input-group-flat">
                                 <input autocomplete="new-password" id="password" name="_password" type="password" tabindex="20" class="form-control" placeholder="{{ 'Password'|trans({}, 'TablerBundle') }}" required="required">
+                                {% block password_show %}
+                                <span class="input-group-text">
+                                    <a
+                                            href="#" class="link-secondary"
+                                            data-bs-toggle="tooltip" title="{{ 'Show password'|trans({}, 'TablerBundle') }}"
+                                            onclick='toggleShowPasswordInput()'
+                                    >
+                                        <div>
+                                            <!-- Download SVG icon from http://tabler.io/icons/icon/eye -->
+                                            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="icon icon-1">
+                                               <path d="M10 12a2 2 0 1 0 4 0a2 2 0 0 0 -4 0"></path>
+                                               <path d="M21 12c-2.4 4 -5.4 6 -9 6c-3.6 0 -6.6 -2 -9 -6c2.4 -4 5.4 -6 9 -6c3.6 0 6.6 2 9 6"></path>
+                                            </svg>
+                                        </div>
+
+                                        <div hidden>
```

Comment:
There is a `hidden` attribute since 10+ years and I never heard about it? I can't believe it.... thanks for using it here 👍  will use it in the future.

Reply:
This what the JS `hidden` property changes 😉 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden